packages/tablev2/tablev2.test.tsx

```diff
@@ -237,7 +237,7 @@ describe("Table v2", () => {
   describe("Sorting", () => {
     it("calls the function passed to the 'sorter' prop when clicking the table headings", () => {
       const sorterFn = jest.fn();
-      const component = mount(
+      render(
```

Comment:
```suggestion
      const { getByTestId } = render(
```

There may be some examples with screen but I think it's a cleaner implementation to set the const here with the render. and then use it below with the event firing.

Reply:
@nataliepina I found this amazing article by Kent( Owner of react testing library ) https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen
where he states otherwise. I think we should use the screen to query elements moving forward.

Reply:
I have also read that one! It sparked a discussion on a previous PR even. We talked about it on the UI team and decided to opt for the render queries. They are nearly the same we've just gone with the latter for consistency as the initial bulk refactor of our tests used mostly render queries. We could consider refactoring down the line to use screen if there is a solid reason to other than the man KCD says to 😄 
Happy to hear out if there are any other strong reasons to use screen instead if you have any thoughts! 

I had a read through this one before option for consistency - https://stackoverflow.com/questions/64792767/what-is-the-difference-between-getbytext-and-screen-getbytext-in-rtl